### PR TITLE
a11y: make SR read field descriptions on widgets

### LINKED ISSUE #2063: [Accessibility][Name, Role, Value - Property Pane_Ask a question (Bot Asks)]: Ensures all ARIA attributes have valid values for prompt edit field.
**User Experience:**
This issue should be resolved for good coding practice as all ARIA attributes have valid values for the edit fields.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
5. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
6. Navigate to property pane appeared at right side of the screen and select 'Bot Asks'.
7. Run Accessibility insight and observe the issue.

**Actual:** 
Edit fields such as prompt, Max turn count, Default value, Allow interruption, Always prompt edit fields doesn't have valid ARIA attributes values.

**Expected:**
Every edit field must have valid aria attributes.

**Note:**
1. Same issue found at "Property Pane_Ask a question (User Asks)".
2. Same issue found at "Property Pane_Ask a question (Other)".
3. Same issue observed at various property panes like Adaptive dialog and other property panes.

**Observation:**
This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**MAS Impacted:** [MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
MAS4 1 1_Property Pane_Ask a question (User Input)
MAS4 1 1_Property Pane_Ask a question (Other)
MAS4 1 1_Property Pane_Ask a question (Bot Asks)

### LINKED ISSUE #2115: [Accessibility][Screen reader - Property Pane_Ask a question (Bot Asks)]: Incorrect role is announced for ? as button in property pane.
**User Experience** 
User who rely on screen reader will get confused if incorrect role is announced by the screen reader.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to '?' just beside of 'Prompt.
6. Observe the issue.

**Actual:** 
On getting the focus, narrator announces role as button along with the descriptive information.

**Expected:**
As it has no functionality, the narrator shouldn't announce the control as button.

**Note:**
Same issue is observed for all the Property Pane which have "?".

**MAS Reference:** [MAS4.2.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
[MAS4.1.2_incorrect role is defined.zip](https://github.com/microsoft/BotFramework-Composer/files/4261888/MAS4.1.2_incorrect.role.is.defined.zip)
MAS4 1 2_Incorrect role is defined for question mark button

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This replaces the IconButton in WidgetLabel.tsx with an ordinary Icon. The properties were copied over for the most part, except for some needed adjustment to the margins. One odd note here is that the screen reader's focus rectangle goes onto the editor field when reading the help text instead of the icon itself, but I think the behavior is basically correct as far as the SR experience goes.

## Task Item

Closes #2115
Closes #2063

## Screenshots

image

### CODE REVIEW

File: Composer/packages/extensions/obiformeditor/src/Form/widgets/WidgetLabel.tsx [hunk 1]
```diff
@@ -50,17 +50,25 @@ const DescriptionCallout: React.FC<DescriptionCalloutProps> = props => {
...
+            color: NeutralColors.gray160,
+            fontSize: '12px',
+
+            marginBottom: '-10px',
+            marginLeft: '4px',
+          },
         }}
-        aria-labelledby={`${id}-description`}
+        aria-label={
+          // append a semicolon so the screen-reader pauses instead of making a confusing run-on
```
Comment (corinagum) [inline Composer/packages/extensions/obiformeditor/src/Form/widgets/WidgetLabel.tsx]: 🤩

**Comment by beyackle** [comment]
Just failing a unit test. I'll fix that and get it back up.

**Comment by beyackle** [comment]
It looks like this PR fixes another issue too (#2063), so I'll link that here too.

File: Composer/packages/extensions/obiformeditor/src/Form/widgets/WidgetLabel.tsx (hunk 1 above)
Comment (a-b-r-o-w-n) [reply Composer/packages/extensions/obiformeditor/src/Form/widgets/WidgetLabel.tsx]: Neat trick.

**Comment by beyackle** [comment]
Okay, verified that these aren't keyboard-navigable now. I'll fix that quickly and put up another version.

**Comment by beyackle** [comment]
Still not 100% happy with the SR experience here, but I think this addresses all the issues.

**Comment by beyackle** [comment]
The original ticket specifically complained about the SR reading "button" when there isn't anything to click on, so we can't leave it a button.

**Comment by a-b-r-o-w-n** [comment]
Maybe the Fabric team has some guidance on this? I'll send you a link to their channel.

**Pull request merged**